src/restc.erl

```diff
@@ -214,13 +213,13 @@ default_content_type(Type) ->
   {<<"content-type">>, get_ctype(Type)}.
 
 do_request(post, Type, Url, Headers, Body, Options) ->
-  Body2 = encode_body(Type, Body),
+  Body2 = restc_body:encode(Type, Body),
```

Comment:
Maybe a name like `Encoded` is just a little nicer on the eyes.

Reply:
Changed to `EncodedBody`